app/services/booking_bug_connection.rb

```diff
@@ -0,0 +1,61 @@
+require 'faraday'
+require 'faraday_middleware'
+
+class BookingBugConnection
```

Comment:
It is a component that at this stage is used by the service, it might be better called BookingBug::Connnection but that did not feel right. 

I think this is as good a place as any for this object at the moment. Once we add a second connection object we should think more about the best location for it. 

Happy to move if you have strong feelings, otherwise I will mark this for future consideration.